controllers/grid/settings/reviewForms/form/PreviewReviewForm.inc.php

```diff
@@ -56,7 +56,7 @@ function initData() {
 			$request = Application::get()->getRequest();
 			$context = $request->getContext();
 			$reviewFormDao = DAORegistry::getDAO('ReviewFormDAO'); /* @var $reviewFormDao ReviewFormDAO */
-			$reviewForm = $reviewFormDao->getById($this->reviewFormId, Application::getContextAssocType(), $context->getId());
+			$reviewForm = $reviewFormDao->getById($this->reviewFormId, Application::getContextAssocType(), $context->getId()); /** @var ReviewForm $reviewForm  */
```

Comment:
Just curious, is this annotation necessary for your IDE, or is for PHP Scrutinizer or something else? I've found that the `DAORegistry::getDAO` annotation (for example) is necessary because the function only specifies `@return DAO` but the calling code wants to execute functions defined only in subclasses. In this case, `ReviewFormDAO::getById` declares `@return ReviewForm`, so I wouldn't expect it to be a problem.

Reply:
@asmecher you are right. 

The problem with that line was the line 58. My editor (Visual Studio Code) supports PHPDoc annotation for those thinks (I may be able to extend it somehow but have not found the time to do that yet). The annotation `/* @var $reviewFormDao ReviewFormDAO */` is not getting parsed as a `variable annotation` but just as another comment. (it needs the `/**` in front instead of `/*`). So the `getById` couldn't get resolved as `returns ReviewForm`. 

In any case, the `/** @var ReviewForm $reviewForm  */` is unnecessary and should be removed.

Reply:
Ah! Great, let's standardize on `/**` -- everything seems to accept that. I have some work to do on annotations for Scrutinizer and hopefully it'll improve things for Visual Studio Code as well.

Reply:
If you like I can "fix" things if I encounter them as well. Or batch-fix them through the code. 
Right now I have found 2 things: 
- the `/*` annotation.
- And a warning about the classes usage in Services: if you try to use `Publication` when annotating, it warns that it could be `\Publication` that I need to consider instead. It's clever enough to work with both, but it warns anyway

Reply:
Either way is fine, and these improvements are good to make to the `master` branch! I tend to do this kind of work when I'm not able to focus on bigger things, but that's just me.